patches/node/avoid_calling_deprecated_method.patch

```diff
@@ -0,0 +1,22 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Samuel Attard <samuel.r.attard@gmail.com>
+Date: Thu, 13 Feb 2020 10:34:31 -0800
+Subject: Avoid calling deprecated method
+
+The {SetExpectInlineWasm} method is deprecated and non-functional since
+V8 v8.1.
+Thus node should stop calling it, so that it can be fully removed in a
+future v8 version.
```

Comment:
Presumably this has already been PR'd to Node?

Reply:
This was cherry-pick'd from https://github.com/electron/electron/pull/22064/commits/7ed51ee3ad22d16e530b3c8ea7a15075e2ec3689.  I don't think its upstreamed yet.

/cc @codebytere